frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/AddAggregationButton/AddAggregationButton.jsx

```diff
@@ -15,6 +15,10 @@ const propTypes = {
   shouldShowLabel: PropTypes.boolean,
 };
 
+const defaultProps = {
```

Comment:
`defaultProps` will eventually be deprecated according to the react devs. Since this is conveniently a functional component let's just default the prop in-line: `export const AddAggregationButton = ({ query, shouldShowLabel = false }) => {`